.github/workflows/build.yaml

```diff
@@ -4,11 +4,13 @@ on:
   push:
     branches:
       - release
+      - master
```

Comment:
super super minor nit: order the branch names alphabetically for easier maintainability